_bioinfcomputing/access_methods.md

```diff
@@ -1,5 +1,210 @@
+
+puTTy is the most widely used secure shell software for Windows. You can install puTTy through "Software Center" and then simply connect to host name `rhino`.
+
+### Mac OS
+
+The default Mac terminal is a good choice for connecting to and using SciComp Linux-based resources.  We also recommend using [iTerm2](https://www.iterm2.com/) for your local terminal program on your Mac.  It has a richer feature set that comes in handy as your computing needs grow.
+
+The only X11 client for Mac is XQuartz. Install XQuartz before running any Linux X11 apps such as RStudio or Matlab. Go [here](http://xquartz.macosforge.org), and download and install the latest version of XQuartz.
+
+After installing XQuartz just start the app. The XQuartz icon will appear in the toolbar showing that it is running. XQuartz is now running an X11 client and that is the only interaction that you need to do with XQuartz. After XQuartz is running, open the Mac terminal. The latest version of XQuartz sets the DISPLAY environment with the default Mac terminal window. We do not recommend using the terminal that comes with XQuartz.
+
+When connecting to a `rhino` with `ssh` always use the -X flag to forward your Xsession to your Mac.
+
+```
+ssh -X HutchID@rhino2
+```
+
+After connecting to a `rhino` verify that your X11 client is working by typing `xeyes`.
+
+
+### Linux
+
+Terminator is a convenient terminal emulator that can have multiple terminals in a single window. You can use Terminator on your Linux desktop. Windows and Mac users will need a X11 software to use SciComp resources as graphical desktop. We recommend [NoMachine NX](https://teams.fhcrc.org/sites/citwiki/SciComp/Pages/NX%20Client%20for%20Windows.aspx) to connect to a Linux machine with Terminator.  
+
+After starting Terminator you can right click on the terminal window and either split it horizontally or vertically. This 1 min video shows you how it works:
+
+[![Terminator](https://img.youtube.com/vi/OmkkbvH_JZU/0.jpg)](https://youtu.be/OmkkbvH_JZU "See how Terminator works")
+
+
+
+## SSH Connections
+SSH uses the TCP network protocol to make connections between systems. TCP is a persistent, stateful connection that is resilient to network interruptions. When you use SSH to run a program on a remote system (even a shell) the connection you make is tied to an SSH process on the remote machine, and all programs are children of that SSH process. If that process stops, all children stop as well. If the network connection does drop between your client and the server, your SSH process stops, and all your running programs stop.
+
+There are several ways around this:
+
+* NoMachine - you can use the NoMachine service
+* GNU Screen or tmux - these utilities run a separate process on the remote host that is not tied to your SSH connection
+* Cluster jobs - you can run your programs using our [slurm cluster](/bioinfcomputing/compute_jobs/), which queues and executes programs independently of your SSH connection
```

Comment:
add something explicit about the fact that this is best for non-interactive tasks (like a script that runs from start to finish).   Newbs need this.